src/Microsoft.Extensions.Options/Options.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Microsoft.Extensions.Options
+{
+    public static class Options
+    {
+        public static IOptions<TOptions> Create<TOptions>(TOptions options) where TOptions : class, new()
+        {
+            return new Options<TOptions>(options);
+        }
+    }
+
+    public class Options<TOptions> : IOptions<TOptions> where TOptions : class, new()
```

Comment:
Or we actually could keep this class private too right?  The factory method is all they need